src/elements/content-sidebar/withSidebarAnnotations.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+import * as React from 'react';
+import getProp from 'lodash/get';
+import noop from 'lodash/noop';
+import type { ContextRouter } from 'react-router-dom';
+import { getBadUserError } from '../../utils/error';
+import type { WithAnnotatorContextProps } from '../common/annotator-context';
+import type { BoxItem, User } from '../../common/types/core';
+
+type Props = {
+    currentUser?: User,
+    file: BoxItem,
+    fileId: string,
+    isOpen: boolean,
+    onVersionChange?: Function,
+} & WithAnnotatorContextProps &
+    ContextRouter;
+
+export default function withSidebarAnnotations(
+    WrappedComponent: React.ComponentType<Props>,
+): React.ComponentType<Props> {
+    class WithSidebarAnnotations extends React.Component<Props> {
```

Comment:
Does this need to be a class? If it was a function component, we could remove `withAnnotatorContext` from the parent and use the `useContext` hook, instead. That may be too large of a rework for this PR, though.

Reply:
I'll look into converting it to a functional component in a follow up PR